src/index.ts

```diff
@@ -233,6 +233,7 @@ function mergeParsers(originalParser: prettier.Parser, parserName: string) {
 }
 
 export { options, parsers, defaultOptions };
+export declare type Options = Partial<JsdocOptions>;
```

Comment:
nitpick: is `declare` necessary? Does it work with just `export type`?

Reply:
> nitpick: is `declare` necessary? Does it work with just `export type`?

yes, `export type` works too. I think that `export type` is better.